tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -220,21 +231,25 @@ public TChannel build() {
 
         private Bootstrap bootstrap(TChannel topChannel) {
             return new Bootstrap()
-                    .group(this.childGroup)
-                    .channel(NioSocketChannel.class)
-                    .handler(this.channelInitializer(false, topChannel))
-                    .validate();
+                .group(this.childGroup)
+                .channel(NioSocketChannel.class)
+                .handler(this.channelInitializer(false, topChannel))
+//                .option(ChannelOption.WRITE_BUFFER_HIGH_WATER_MARK, 32 * 1024)
```

Comment:
it can throw exceptions when used. We need to be able to handle exceptions first.